build-tests/api-documenter-test/etc/api-documenter-test.api.json

```diff
@@ -166,7 +166,8 @@
         "@preapproved": true,
         "@docCategory": true,
         "@myCustomTag": true
-      }
+      },
+      "reportUnsupportedHtmlElements": false
```

Comment:
Why'd this get disabled everywhere? We want these to be reported.

Reply:
I'm not sure where these came from but I'm happy to remove the changes. Is this something that was programmatically altered from build/update/install commands?

Reply:
Shouldn't have been.